doc/modules/model_evaluation.rst

```diff
@@ -1257,29 +1257,31 @@ implements label ranking average precision (LRAP). This metric is linked to
 the :func:`average_precision_score` function, but is based on the notion of
 label ranking instead of precision and recall.
 
-Label ranking average precision (LRAP) is the average over each ground truth
-label assigned to each sample, of the ratio of true vs. total labels with lower
-score. This metric will yield better scores if you are able to give better rank
+Label ranking average precision (LRAP) averages over the samples the answer to
+the following question: for each ground truth label, what fraction of higher-ranked
+labels were true labels? ("Higher-ranked" means that the algorithm assigned a higher
+score :math:`\hat{f}_{ik}` and a lower value of
+:math:`\text{rank}_{ik} = \left|\left\{l: \hat{f}_{il} \geq \hat{f}_{ik} \right\}\right|`.)
```

Comment:
I think providing notation to define rank is a bit excessive in this context. The definition is also repeated below in a more notational context. I appreciate that you are trying to be precise, but readability is important here too.

Reply:
OK, I'll remove the parenthetical sentence

> ("Higher-ranked" means that the algorithm assigned a higher
>  +score :math:`\hat{f}_{ik}` and a lower value of
>  +:math:`\text{rank}_{ik} = \left|\left\{l: \hat{f}_{il} \geq \hat{f}_{ik} \right\}\right|`.)